operator/cmd/syncclusterconfig/sync_test.go

```diff
@@ -204,23 +201,52 @@ func TestSync(t *testing.T) {
 		},
 		{
 			// In declarative mode, we'll drop any unspecified configuration
-			Config:          map[string]any{},
-			UsersTXTDir:     os.TempDir() + "/this-path-does-not-exist",
-			DeclarativeMode: true,
-			Expected:        map[string]any{},
+			Config:      map[string]any{},
+			UsersTXTDir: os.TempDir() + "/this-path-does-not-exist",
+			Mode:        SyncerModeDeclarative,
+			Expected:    map[string]any{},
 		},
 		{
-			// Even in declarative mode, the implicit settings from UsersTXT
-			// should still be applied.
+			// Even in declarative mode, the UsersTXT
+			// is still applied.
+			Mode:        SyncerModeDeclarative,
+			UsersTXTDir: filepath.Dir(usersTXTPath),
 			Config: map[string]any{
-				"superusers": []string{user},
+				"superusers": []string{admin},
 			},
-			UsersTXTDir:     filepath.Dir(usersTxtYAMLPath),
-			DeclarativeMode: true,
 			Expected: map[string]any{
 				"superusers": []any{admin, user},
 			},
 		},
+		{
+			Mode: SyncerModeDeclarative,
+			Config: map[string]any{
+				// An aliased config value.
+				"schema_registry_normalize_on_startup": true,
+			},
+			Expected: map[string]any{
+				"schema_registry_always_normalize": true,
```

Comment:
it might be worthwhile to add in a second call to `cmd.ExecuteContext` in this test, I'm pretty sure that a second execution of the command will result in this getting removed without the aliasing normalization